src/general/requiredlabels/constraint.tmpl

```diff
@@ -38,5 +38,9 @@ spec:
                       the regular expression.
   targets:
     - target: admission.k8s.gatekeeper.sh
+      code:
+      - engine: K8sNativeValidation
+        source:
+{{ file.Read "src/general/requiredlabels/src.cel" | strings.Indent 10 | strings.TrimSuffix "\n" }}
```

Comment:
should add a validation for src.cel: https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/generate.sh#L39

Reply:
will it not error out for policies that sill do not have `src.cel` file? May be we probably should add `src.cel` to the list once we have cel source for all policies. Even in that case the referential policies won't have `src.cel`. wdyt?

Reply:
https://github.com/open-policy-agent/gatekeeper-library/blob/883683a0a49f82dea424f4a46cbcf9c94659c0c9/scripts/generate.sh#L39

if there's no src.cel file, it won't go into the loop.

Reply:
So by validation of the `src.cel` file do you mean syntax validation? will it run or not kind of a thing?

Reply:
the linked code  https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/generate.sh#L39 checks if the file exists or not.

Reply:
> I was suggesting to create a new loop. if there's no src.cel file, it won't go into the loop.

@ritazh I am sorry for the confusion (may be I am getting something wrong here). Seems like what you are suggesting is a new for loop that verifies the existance of `src.cel` file. What happens when the file do no exists? 

IMO I do not think we need this validation loop at all (`for req in src.rego src_test.rego constraint.tmpl; do`) because if those files are missing we will error out during generation of the template - [generate.sh#l16](https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/generate.sh#L16).

I think we should remove the validation code.

Reply:
>  if those files are missing we will error out during generation of the template

sgtm

Let's leave the existing loop since it also checks for src_test.rego otherwise it will only break when running the tests.